libcasr/src/java.rs

```diff
@@ -0,0 +1,272 @@
+        assert_eq!(stacktrace[0].debug.line, 388);
+        assert_eq!(
+            stacktrace[0].function,
+            "java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open".to_string()
+        );
+        assert_eq!(stacktrace[1].debug.file, "NativeLibraries.java".to_string());
+        assert_eq!(stacktrace[1].debug.line, 232);
+        assert_eq!(
+            stacktrace[1].function,
+            "java.base/jdk.internal.loader.NativeLibraries.loadLibrary".to_string()
+        );
+        assert_eq!(stacktrace[2].debug.file, "NativeLibraries.java".to_string());
+        assert_eq!(stacktrace[2].debug.line, 174);
+        assert_eq!(
+            stacktrace[2].function,
+            "java.base/jdk.internal.loader.NativeLibraries.loadLibrary".to_string()
+        );
+        assert_eq!(stacktrace[3].debug.file, "NativeLibraries.java".to_string());
+        assert_eq!(stacktrace[3].debug.line, 315);
+        assert_eq!(
+            stacktrace[3].function,
+            "java.base/jdk.internal.loader.NativeLibraries.findFromPaths".to_string()
+        );
+        assert_eq!(stacktrace[4].debug.file, "NativeLibraries.java".to_string());
+        assert_eq!(stacktrace[4].debug.line, 287);
+        assert_eq!(
+            stacktrace[4].function,
+            "java.base/jdk.internal.loader.NativeLibraries.loadLibrary".to_string()
+        );
+        assert_eq!(stacktrace[5].debug.file, "ClassLoader.java".to_string());
+        assert_eq!(stacktrace[5].debug.line, 2422);
+        assert_eq!(
+            stacktrace[5].function,
+            "java.base/java.lang.ClassLoader.loadLibrary".to_string()
+        );
+        assert_eq!(stacktrace[6].debug.file, "Runtime.java".to_string());
+        assert_eq!(stacktrace[6].debug.line, 818);
+        assert_eq!(
+            stacktrace[6].function,
+            "java.base/java.lang.Runtime.loadLibrary0".to_string()
+        );
+        assert_eq!(stacktrace[7].debug.file, "System.java".to_string());
+        assert_eq!(stacktrace[7].debug.line, 1989);
+        assert_eq!(
+            stacktrace[7].function,
+            "java.base/java.lang.System.loadLibrary".to_string()
+        );
+        assert_eq!(
+            stacktrace[8].debug.file,
+            "ExampleFuzzerNative.java".to_string()
+        );
+        assert_eq!(stacktrace[8].debug.line, 20);
+        assert_eq!(
+            stacktrace[8].function,
+            "ExampleFuzzerNative.<clinit>".to_string()
+        );
+        assert_eq!(stacktrace[9].debug.file, "Class.java".to_string());
+        assert_eq!(stacktrace[9].debug.line, 0);
+        assert_eq!(
+            stacktrace[9].function,
+            "java.base/java.lang.Class.forName".to_string()
+        );
+        assert_eq!(stacktrace[10].debug.file, "Reproducer.java".to_string());
+        assert_eq!(stacktrace[10].debug.line, 24);
+        assert_eq!(stacktrace[10].function, "Reproducer.main".to_string());
+    }
+}
```

Comment:
Maybe we could cover with tests others parsing functions in lib such as `parse_exception` too?

Reply:
`parse_exception` isn't complex and isn't covered with tests for other languages. Maybe this could be done in another PR.